frontend/lib/norent/start-account-or-login/tests/steps.test.tsx

```diff
@@ -63,7 +63,7 @@ describe("start-account-or-login flow", () => {
     pal.ensureLinkGoesTo(/back/i, "/welcome");
     pal.clickButtonOrLink(/privacy/i);
     pal.ensureLocation(routes.phoneNumberTermsModal);
-    pal.rr.getByText(/anonymized data/i);
+    pal.rr.getByText("justfix.privacyInfoModalText");
```

Comment:
This is kind of weird--basically, because lingui's babel transforms are in-place yet the localization JS bundle isn't being loaded, lingui doesn't know what to substitute for the message with id "justfix.privacyInfoModalText" so it's just giving the id (normally this isn't weird because the id _is_ the english version of the text).  However I think that verifying the id is there is actually less brittle than testing to see if certain characters (like "anonymized data") are in the content, since the content might eventually change, but the id won't... Let me know if you disagree though!